dex-it/src/test/scala/com/wavesplatform/it/sync/PostgresHistoryDatabaseTestSuite.scala

```diff
@@ -210,6 +212,17 @@ class PostgresHistoryDatabaseTestSuite extends MatcherSuiteBase with HasPostgres
     ctx.run(querySchema[EventRecord]("events").delete)
   }
 
+  "Node should provide its name to postgres" in {
+
+    val buyOrder = mkOrder(bob, wavesBtcPair, BUY, 270477189L, 28259L)
```

Comment:
amount and price values could be written like (e.g. 1.waves, or 10.btc)

Reply:
Fixed. Also changed it in the next test case